revolv/payments/services.py

```diff
@@ -13,12 +15,14 @@ class PaymentServiceException(Exception):
 class PaymentService(object):
 
     @classmethod
-    def create_payment(cls, user, amount, payment_instrument):
+    def create_payment(cls, user, entrant, amount, project, payment_instrument):
```

Comment:
True - I don't know if that's worth it in this case but in general it's a code smell when a method has a bunch of parameters
http://blog.codinghorror.com/code-smells/
http://stackoverflow.com/questions/174968/how-many-parameters-are-too-many